codelists/actions.py

```diff
@@ -58,6 +58,7 @@ def create_or_update_codelist(
     methodology=None,
     references=None,
     signoffs=None,
+    force_create=False,
```

Comment:
In a function called `create_or_update_codelist`, a parameter called `force_create` is a bit confusing.  (For instance: if it's true, are we ensuring that we create a new codelist instead of updating an existing one?)

I'd suggest calling it something like `always_create_new_version`.